components/opc-ua/src/Microsoft.Azure.IIoT.OpcUa.Protocol/src/Runtime/Default/ClientServicesConfig.cs

```diff
@@ -63,11 +62,12 @@ public class ClientServicesConfig : ConfigBase, IClientServicesConfig, ISecurity
         /// <inheritdoc/>
         public bool RejectSha1SignedCertificates => _security.RejectSha1SignedCertificates;
```

Comment:
One could use this option.  But I would default to false and as per discussion minimize config surface.

Reply:
it is defaulted to false, anywhere except Gateway Server, (previous comment,), where was already set to true and did not questioned the reason :)